packaging/systemd/microshift.service

```diff
@@ -9,6 +9,15 @@ ExecStart=microshift run
 Restart=always
 User=root
 Type=notify
+Delegate=yes
+CPUAccounting=yes
+BlockIOAccounting=yes
+MemoryAccounting=yes
+# Having non-zero Limit*s causes performance problems due to accounting overhead
+# in the kernel. We recommend using cgroups to do container-local accounting.
+LimitNOFILE=infinity
+LimitNPROC=infinity
+LimitCORE=infinity
```

Comment:
Why are we resetting these 3 specific limits? How do we know we should not bump more to avoid the mentioned performance problems.

Reply:
Also, can we add a link in the comment that describes the performance problem?

Reply:
I'm not sure I follow @ggiguash , are you talking about the performance issues caused by a very high value in LimitNOFILE? I could reduce it to 1048576 as it seems to be the standard for other project.

Reply:
https://github.com/coreos/fedora-coreos-tracker/issues/329

Reply:
`LimitNOFILE = 1048576` seems to be the standard number for most of the projects that deal with containers from a systemd service.

Reply:
Can we update the comment to something more concrete? I think it's important to note what part of the kernel is not performing well.
```
Having higher limit values cause file system performance problems due to accounting overhead in the kernel. 
We recommend using cgroup to do container-local accounting.
```